### PR TITLE
Exception and validation

### PR DESCRIPTION
**Pull request by BohdanS-UA**

### CODE REVIEW

File: src/main/java/mate/academy/bookstorev2/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
+import mate.academy.bookstorev2.validation.UniqueIsbn;
 
 @Getter
 @Setter
 public class CreateBookRequestDto {
+    @NotNull
```
Comment (Vasul-Pankiv) [inline src/main/java/mate/academy/bookstorev2/dto/CreateBookRequestDto.java]:
```suggestion
    @NotBlank
```

File: src/main/java/mate/academy/bookstorev2/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
     private String title;
+    @NotNull
```
Comment (Vasul-Pankiv) [inline src/main/java/mate/academy/bookstorev2/dto/CreateBookRequestDto.java]: same here

**Review by Vasul-Pankiv — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/bookstorev2/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
 @Setter
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @PositiveOrZero
+    @NotNull
     private BigDecimal price;
+    @NotNull
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/bookstorev2/dto/CreateBookRequestDto.java]:
```suggestion
    @NotBlank
```

File: pom.xml [hunk 4]
```diff
@@ -65,6 +65,17 @@
             <artifactId>liquibase-maven-plugin</artifactId>
             <version>${liquibase.version}</version>
         </dependency>
+
+        <dependency>
```
Comment (Elena-Bruyako) [inline pom.xml:69]: It's better to add this 
```suggestion
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-validation</artifactId>
        </dependency>
```

File: pom.xml [hunk 5]
```diff
@@ -65,6 +65,17 @@
...
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-validator</artifactId>
```
Comment (Elena-Bruyako) [inline pom.xml]: remove

File: pom.xml [hunk 6]
```diff
@@ -65,6 +65,17 @@
...
+            <version>8.0.1.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator-annotation-processor</artifactId>
```
Comment (Elena-Bruyako) [inline pom.xml]: remove

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's fix some comments

**Review by Vasul-Pankiv — APPROVED** [review]

**Pull request merged**